internal/incoming/odfi/scheduler.go

```diff
@@ -188,13 +188,14 @@ func (s *PeriodicScheduler) tick(shard *service.Shard) error {
 }
 
 func (s *PeriodicScheduler) alertOnError(err error) {
-	if s == nil || s.alerter == nil {
+	if s == nil || len(s.alerters) == 0 {
```

Comment:
We don't need these length checks since we've got `Alterers`

```suggestion
	if s == nil {
```